tdnf-cli-libs.pc.in

```diff
@@ -6,6 +6,6 @@ includedir=@includedir@/tdnf
 Name: tdnf-cli-libs
 Description: tdnf cli libs
 Version: @VERSION@
-Requires:
+Requires: tdnf
```

Comment:
cli libs should not require tdnf.

Reply:
cli libs require libtdnf, which is what the `tdnf` pkgconfig module represents.

Actually, this pkgconfig module is poorly named, but I left it alone for now. It should be named something like `tdnfcli` or `libtdnfcli` instead. The `tdnf` pkgconfig module should probably be `libtdnf` instead.

But I left this alone for now because I wasn't sure if you were using pkgconfig for your software.